panel/src/panel/content.js

```diff
@@ -133,6 +133,9 @@ export default (panel) => {
 				return;
 			}
 
+			// update the last modification timestamp
+			panel.view.props.lock.modified = new Date();
```

Comment:
`Todo:` I think this is in the wrong place. This needs to go into `save()` in the `finally` section doesn't it?